src/main/java/seedu/taassist/storage/JsonAdaptedStudent.java

```
@@ -28,21 +28,21 @@ class JsonAdaptedStudent {
     private final String phone;
     private final String email;
     private final String address;
-    private final List<JsonAdaptedTag> tagged = new ArrayList<>();
+    private final List<JsonAdaptedModuleClass> classes = new ArrayList<>();
 
     /**
      * Constructs a {@code JsonAdaptedStudent} with the given student details.
      */
     @JsonCreator
     public JsonAdaptedStudent(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
             @JsonProperty("email") String email, @JsonProperty("address") String address,
-            @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+            @JsonProperty("tagged") List<JsonAdaptedModuleClass> classes) {
```

Should the property `"tagged"` be renamed too?

---

I think we can create another issue to handle this, since this involves changing all the tagging in the dummy data. Missing anything out here can cause errors that prevent merging of this commit.